test/unit/models/users_model.test.js

```diff
@@ -0,0 +1,39 @@
+const { MongoClient } = require('mongodb');
+const rewire = require('rewire');
+const { url, dbName } = require('../../../config');
+
+const usersModel = rewire('../../../src/models/users_model');
+/* eslint-disable no-underscore-dangle */
+const storeUser = usersModel.__get__('storeUser');
```

Comment:
using rewire to test private methods of the user_model.js

Reply:
👍